docs/sources/features/datasources/stackdriver.md

```diff
@@ -133,7 +133,7 @@ Group by resource or metric labels to reduce the number of time series and to ag
 
 #### Meta data labels
 
-Meta data labels are only returned in the time series response if they're present in the request. There's no API for retrieving meta data labels, and therefore it's not possible to populate the group by dropdown with the meta data labels that are avaiable for the selected service and metric. However, the group by field dropdown comes with a pre-defined list of common system labels. User labels cannot be pre-defined, but it's possible to enter them manually in the group by field. If a meta data label, user label or system label, is included in the group by segment, it will be possible to create filters based on it and to expand its value on the alias field.
+Meta data labels are only returned in the time series response if they're part of the group by segment in the time series request. There's no API for retrieving meta data labels, and therefore it's not possible to populate the group by dropdown with the meta data labels that are avaiable for the selected service and metric. However, the group by field dropdown comes with a pre-defined list of common system labels. User labels cannot be pre-defined, but it's possible to enter them manually in the group by field. If a meta data label, user label or system label, is included in the group by segment, it will be possible to create filters based on it and to expand its value on the alias field.
```

Comment:
```suggestion
Metadata labels are only returned in the time series response if they're part of the group by segment in the time series request. There's no API for retrieving metadata labels, so it's not possible to populate the group by dropdown with the metadata labels that are available for the selected service and metric. However, the group by field dropdown comes with a pre-defined list of common system labels. 

User labels cannot be pre-defined, but it's possible to enter them manually in the group by field. If a metadata label, user label or system label is included in the group by segment, then you can create filters based on it and expand its value on the alias field.
```

Reply:
A few more things:

It looks like you might be talking about field names in some cases. If so, capitalize them to match what is on the screen and make them bold, per the documentation style guide.

Most of this paragraph seems to assume the reader understands the API and the metadata labels, which I unfortunately do not, so it is a bit dense for me to clearly understand what you are trying to say here.